ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/minimal-permission/cucushift-installer-rehearse-gcp-ipi-minimal-permission-workflow.yaml

```diff
@@ -5,10 +5,14 @@ workflow:
     - chain: cucushift-installer-rehearse-gcp-ipi-minimal-permission-provision
     post:
     - chain: cucushift-installer-rehearse-gcp-ipi-minimal-permission-deprovision
+    env:
+      CONTROL_PLANE_NODE_TYPE: "custom-4-16384"
```

Comment:
The custom machine type need to be pre-created ?

Reply:
@jianlinliu No. 

_FYI For example, with "custom-4-16384", the first number "4" specifies the number of vCPUs, and the  second number "16384" specifies the memory size (MB)._

Reply:
Good to know.